aioshelly/__init__.py

```diff
@@ -106,9 +112,9 @@ def __init__(
         self.coap_context = coap_context
         self.aiohttp_session = aiohttp_session
         self.options = options
-        self.d = None
+        self.coap_d = None
```

Comment:
This is also a breaking change.

Reply:
Please list them in the README and they will need to be part of the release notes.